Packages/Ethereum/.swiftpm/xcode/xcshareddata/xcschemes/potato.xcscheme

```diff
@@ -51,18 +51,34 @@
          </BuildableReference>
       </BuildableProductRunnable>
       <CommandLineArguments>
+         <CommandLineArgument
+            argument = "VestingPool"
+            isEnabled = "NO">
+         </CommandLineArgument>
+         <CommandLineArgument
+            argument = "SafeToken"
+            isEnabled = "NO">
+         </CommandLineArgument>
+         <CommandLineArgument
+            argument = "Airdrop"
+            isEnabled = "NO">
+         </CommandLineArgument>
          <CommandLineArgument
             argument = "DelegateRegistry"
-            isEnabled = "YES">
+            isEnabled = "NO">
```

Comment:
what does it mean that it is not enabled?

Reply:
It's added but not used because I want to use another one for the next start.